client/index.html

```diff
@@ -10,23 +10,32 @@
     <title>Document</title>
 </head>
 <body>
-    <div class = "container">
```

Comment:
I like that you used a CSS framework.  Would have liked to see some styling on the created posts though.